scripts/seed-data

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
-
 set -e -x
 
 BASE_DIR=${1}
 CLOUD_DATA=${2}
+IFS=',' read -ra FILES <<< "${3}"
```

Comment:
Googled it

On Fri, Apr 10, 2015, 01:40 Darren Shepherd <notifications@github.com>
wrote:

> In scripts/seed-data
> <https://github.com/rancherio/os-installer/pull/11#discussion_r28100396>:
>
> >  set -e -x
> >
> >  BASE_DIR=${1}
> >  CLOUD_DATA=${2}
> > +IFS=',' read -ra FILES <<< "${3}"
>
> impressive bashism
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rancherio/os-installer/pull/11/files#r28100396>.
>